src/components/cards/StakingReturnsCard.tsx

```diff
@@ -29,6 +32,12 @@ export function StakingReturnsCard({ fromCurrencyLogos, toCurrencyLogos, text, s
     )
   }
 
+  const renderEstimatedReturn = () => {
+    if (apy === 0) return null
+    const estimatedReturnMsg = apy > 0 ? toFixed(apy.toString(), 1, 1) + '% APR' : 'N/A'
```

Comment:
Does the `if` statement on the line above mean the `apy > 0 ? ` ternary is no longer needed? Maybe the `if` statement could be changed to `if (apy <= 0) return null`, to hit both cases.

Reply:
Correct, fixed.